standard/tokens/nft/how-it-works.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: "NFT: How it works"
+sidebarTitle: "How it works"
+---
+
+import { Aside } from '/snippets/aside.jsx';
+import { Image } from '/snippets/image.jsx';
+
+This article describes the basic ideas and processes behind the implementation of Non‑Fungible Tokens (NFTs) in the TON Blockchain, aligned with [TEP‑62](https://github.com/ton-blockchain/TEPs/blob/1fbc23cac69723c53251f686ec90d81bf0e83443/text/0062-nft-standard.md).
+
+## Related smart contracts
+
+Standardized NFTs on TON are implemented using a set of smart contracts, including:
+
+- NFT collection smart contract
+- NFT item smart contract
+
+The NFT standard provides only a general interaction scheme, leaving specific implementation details to developers.
+
+<Image
+  src="/resources/images/nft/collection_light.svg"
+  darkSrc="/resources/images/nft/collection_dark.svg"
+  alt="NFT collection"
+/>
+
+### NFT collection
+
+The collection is the source of truth for items. It should provide each NFT item's address, its own collection metadata, and, given an index and individual item metadata, can provide full item metadata.
+
+### NFT item
+
+Each NFT item is its own smart contract account ([Explanation of why one item equals one contract](/contract-dev/contract-sharding)). It provides the collection address, index, current owner, and individual metadata. On a valid transfer from the current owner, it updates the owner, optionally notifies the new owner, and returns excess Toncoin to the specified address.
```

Comment:
the `Explanation of why one item equals one contract` is too long and adds parentheses. perhaps rephrase this sentence to mention the contract sharding idea natively.